charts/gardener/controlplane/values.yaml

```diff
@@ -5,7 +5,7 @@ global:
   apiserver:
     enabled: true
     serviceEnabled: true
-    replicaCount: 1
+    replicaCount: 3
```

Comment:
Why is this changed?

Reply:
The Admission Controller gets 3 replicas per default, this does not match the one replica for the API server getting most of the load. I thought it is a sensible default. But I can surely remove it again.

Reply:
Yeah, let's keep it `1` for now. Probably we could make the default for GAC `1` as well to not waste too many resources in small dev/experimental landscapes.